definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-prod.json

```diff
@@ -54,12 +54,5 @@
     "CaseEventID": "EnterAOSOffline_Respondent_fromAosOverdue",
     "UserRole": "caseworker-divorce-courtadmin_beta",
     "CRUD": "R"
-  },
```

Comment:
I think this could remain there. 
Technically there shouldn't be a difference whether it is here or not from a user perspective.

Reply:
I might actually remove all of the other events. The only reason we had them as "R" is that we didn't have feature-toggle before.
This will eliminate complexity when going live.
What do you think @yarpo and @Dizrupto ?

Reply:
Un-did my changes, @Dizrupto